LearnosityDemo/Pages/AuthorAideAPIDemo.cshtml.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Text;
+using LearnositySDK.Request;
+using LearnositySDK.Utils;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Newtonsoft.Json;
+
+namespace LearnosityDemo.Pages
+{
+    public class AuthorAideAPIDemoModel : PageModel
+    {
+        public void OnGet(string mode)
+        {
+            Init init = initializeAide();
+            ViewData["InitJSON"] = init.generate();
+        }
+
+        private static Init initializeAide()
```

Comment:
It is good to use PascalCase for method name i.e. InitializeAide()

Reply:
if you check the codebase, it use camelCase, so I guess we should follow existing